.github/workflows/ci.yml

```diff
@@ -127,12 +127,10 @@ jobs:
         override: true
 
     - name: install python dependencies
-      run: |
-        pip install -U setuptools wheel
-        pip install 'git+https://github.com/joerick/cibuildwheel.git@refs/pull/194/head'
```

Comment:
Yeah, I was looking into this as well. That PR has been merged in March 2020. The CI says it used cibuildwheel v1.2.0 with v1.8.0 being the most recent release.

Reply:
this was a temporary hack before cibuildwheel master/release versions support github actions. Now looks to working well with the latest release.